go/src/vendors/sonic/client/dbclient/counter_repository.go

```diff
@@ -0,0 +1,75 @@
+// Defines APIs to get interface counters from redis db - CountersDB.
+package dbclient
+
+import (
+	"strconv"
+	. "go/src/lib/lomcommon"
```

Comment:
My suggestion is to remote dot in package imports. Use lomcommon to refer package data. If there are conflicts . notation may cause one of the functions to be shadowed and become unavailable.